src/coreclr/inc/profilepriv.inl

```diff
@@ -36,11 +35,1216 @@ inline ProfilerStatus CurrentProfilerStatus::Get()
     return m_profStatus;
 }
 
+inline EventMask& EventMask::operator=(const EventMask& other)
+{
+    m_eventMask = other.m_eventMask;
+    return *this;
+}
+
+inline BOOL EventMask::IsEventMaskSet(DWORD eventMask)
+{
+    return (eventMask == (DWORD)-1) 
```

Comment:
This was an artifact from the initial change I had, I was using -1 to indicate that it should call every profiler regardless of event mask. I am no longer using that concept so it can be removed